.changelog/24073.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+ui: Fixes an issue where variables paths would not let namespaced users write variables unless they also had * namespace variable write permissions
```

Comment:
I don't know for sure but I would bet that this breaks markdown in the changelog once we stitch all the text files together. Let's avoid the problem by writing it out:

```suggestion
ui: Fixes an issue where variables paths would not let namespaced users write variables unless they also had wildcard namespace variable write permissions
```